addons/survey/tests/test_survey_invite.py

```diff
@@ -231,3 +231,26 @@ def test_survey_invite_token_by_email_nosignup(self):
         answers = self.env['survey.user_input'].search([('survey_id', '=', self.survey.id)])
         self.assertEqual(len(answers), 1)
         self.assertEqual(answers.partner_id.display_name, first_partner.display_name)
+
+    @users('survey_manager')
+    def test_survey_invite_email_from(self):
```

Comment:
Could you add it in the test class in an alphabetical order ? As tests are executed in alphabetical order, better order them according to the same ordering in the file, it eases update :)